files/en-us/web/javascript/reference/global_objects/asyncgeneratorfunction/index.html

```diff
@@ -0,0 +1,114 @@
+---
+title: AsyncGeneratorFunction
+slug: Web/JavaScript/Reference/Global_Objects/AsyncGeneratorFunction
+tags:
+- Constructor
+- ECMAScript 2018
+- AsyncGeneratorFunction
+- Iterator
+- JavaScript
+- Reference
+---
+<div>{{JSRef}}</div>
+
+<p>The <strong><code>AsyncGeneratorFunction</code> constructor</strong> creates a new
+  {{jsxref("Statements/async_function*", "async generator function", "", 1)}} object. In JavaScript,
+  every async generator function is actually an <code>AsyncGeneratorFunction</code> object.</p>
+
+<p>Note that <code>AsyncGeneratorFunction</code> is not a global object. It could be obtained
+  by evaluating the following code.</p>
+
+<pre class="brush: js">Object.getPrototypeOf(async function*(){}).constructor
+</pre>
+
+<h2 id="Syntax">Syntax</h2>
+
+<pre
+  class="brush: js">new AsyncGeneratorFunction ([<var>arg1</var>[, <var>arg2</var>[, ...<var>argN</var>]],] <var>functionBody</var>)</pre>
+
+<h3 id="Parameters">Parameters</h3>
+
+<dl>
+  <dt><code>arg1, arg2, ... arg<em>N</em></code></dt>
```

Comment:
Hm, strictly speaking, the commas are not part of the code.
```suggestion
  <dt><code>arg1</code>, <code>arg2</code>, ... <code>arg<em>N</em></code></dt>
```

Reply:
Thanks, will take your suggestion. That should probably be fixed on the original page as well&hellip;

https://github.com/mdn/content/blob/5aa71dc0e462f41519819877a047d2237aeaddee/files/en-us/web/javascript/reference/global_objects/generatorfunction/index.html#L32

Reply:
Yupp. I'd suggest to open a new PR for that, that can be reviewed in parallel.